zkevm-circuits/src/witness/block.rs

```diff
@@ -239,7 +243,8 @@ pub fn block_convert<F: Field>(
     code_db: &bus_mapping::state_db::CodeDB,
 ) -> Result<Block<F>, Error> {
     let rws = RwMap::from(&block.container);
-    rws.check_value();
+    // TODO add me back
+    // rws.check_value();
```

Comment:
```suggestion
    rws.check_value();
```

Reply:
Thanks for catch up this, retain it back in commit https://github.com/privacy-scaling-explorations/zkevm-circuits/commit/0646b790010306d9df41cd27002bf9890d017202